.github/workflows/build-image.yaml

```
@@ -0,0 +1,39 @@
+---
+name: publish container image
+
+on:
+  push:
+    branches:
+      - master
```

https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags
```suggestion
   tags:        
      - '**'  
```

---

If so, we must remove release and master (latest) as it won’t build.  Tags are usually release as well.